ui/js/actions/content.js

```diff
@@ -505,3 +505,36 @@ export function doPublish(params) {
     });
   };
 }
+
+export function doAbandonClaim(claimId, txid, nout) {
+  return function(dispatch, getState) {
+    const state = getState();
+
+    dispatch(doCloseModal());
```

Comment:
This should probably be in the modal not in the action.

Reply:
Yes, I did it.